pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/extensions/filter/BrokerFilterTestBase.java

```diff
@@ -115,6 +116,7 @@ public BrokerLookupData getLookupData(String version) {
         }};
         return new BrokerLookupData(
                 webServiceUrl, webServiceUrlTls, pulsarServiceUrl,
-                pulsarServiceUrlTls, advertisedListeners, protocols, true, true, version);
+                pulsarServiceUrlTls, advertisedListeners, protocols, true, true,
+                ExtensibleLoadManagerImpl.class.getName(), -1, version);
```

Comment:
We should pass loadmanagerClassName as arg and cover brokers with different BrokerLookupData .loadmanagerClassName during rollback <--> deployment.

case new LM deployment:
1. brokers with new LM
2. brokers with old LM

case rollback to old LM:
1. brokers with new LM
2. brokers with old LM

Reply:
Updated.